src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/dto/request/CertificateDocumentRequestDto.kt

```diff
@@ -45,5 +46,7 @@ data class CertificateDocumentRequestDto(
     val trustLevel: String? = null,
     val validator: TrustValidatorDto? = null,
     val uploader: String? = null,
+    val certificateDocumentID: String,
```

Comment:
Instead of certificateDocumentID, i think it will be uniform if we use just documentID.